pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/BasicModelDefTest.java

```diff
@@ -159,9 +159,7 @@ public void parallelPipeline() throws Exception {
 
     @Test
     public void executionModelAction() throws Exception {
-        prepRepoWithJenkinsfile("executionModelAction");
-
-        WorkflowRun b = getAndStartBuild();
+        WorkflowRun b = expect("executionModelAction").go();
         j.assertBuildStatusSuccess(j.waitForCompletion(b));
```

Comment:
`go()` should have already run the build, waited for completion and asserted success.